virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar-widget/launchbar-widget.component.html

```diff
@@ -32,23 +38,50 @@ <h6 class="launchbar-plugin-version">{{getPluginVersion()}}</h6>
             <div class="timeAgo">{{timeSince[i]}}</div>
           </div>
       </div>
-      <hr class="notification-breaker">
+      <hr class="notification-breaker" [style.border-top-color]="color.launchbarMenuText+'1c'">
```

Comment:
How does this work? `+1c`.

Reply:
So same with my previous comment, this adds a level of transparency to the hex color code, in this code 1c is a stronger amount